kbase-extension/static/kbase/config/job_config.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Job vocab moved to a separate file that the front and back end load to ensure that they are both using the same terminology. Ideally we would have some integration tests that faked out ee2 / workspace responses so we could see the FE and BE passing messages back and forth, but in lieu of that...

Reply:
That's a thing we can set up now, I think. It might take a little tinkering, but should be doable with the integration test stack. At least, I think so.